### PR TITLE
Text containing links now appear as hyperlinks in the message view

### PR DESCRIPTION
**Pull request by AnupNair08**

### CODE REVIEW

File: src/views/Message/index.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ export default function Message(props) {
         <div
           className={['bubble', `${isMine ? 'mine' : ''}`].join(' ')}
           title={friendlyTimestamp}>
-          {data.msg}
+          {/* {data.msg} */}
+          <div dangerouslySetInnerHTML={{ __html: data.msg }} />
```
Comment (savannahar68) [inline src/views/Message/index.js:24]: Why dangerouslySetInnerHTML?

File: src/views/Compose/index.js [hunk 2]
```diff
@@ -27,6 +27,19 @@ export default function Compose(props) {
         console.log(err);
       });
   }
+  function urlify(text) {
```
Comment (savannahar68) [inline src/views/Compose/index.js:30]: Please add a working screenshot of text without URL, text with URL and only URl in message section

File: src/views/Message/index.js (hunk 1 above)
Comment (AnupNair08) [reply src/views/Message/index.js:24]: Links found in the message are replaced by textual anchor tag to maintain the way we store messages on the database (string format). dangerouslySetInnerHTML is used for rendering only the markup part of the message.

File: src/views/Compose/index.js (hunk 2 above)
Comment (AnupNair08) [reply src/views/Compose/index.js:30]: sspng

**Pull request merged**